app/src/androidTest/java/com/duckduckgo/app/settings/SettingsViewModelTest.kt

```diff
@@ -147,13 +162,15 @@ class SettingsViewModelTest {
     }
 
     @Test
-    fun whenLightThemeTogglesOffThenDataStoreIsUpdatedAndUpdateThemeCommandIsSent() {
-        testee.onLightThemeToggled(false)
-        verify(mockAppSettingsDataStore).theme = DuckDuckGoTheme.DARK
+    fun whenLightThemeTogglesOffThenDataStoreIsUpdatedAndUpdateThemeCommandIsSent() = coroutineTestRule.runBlocking {
+        testee.commands().test {
+            testee.onLightThemeToggled(false)
+            verify(mockAppSettingsDataStore).theme = DuckDuckGoTheme.DARK
+
+            assertEquals(Command.UpdateTheme, expectItem())
 
-        testee.command.blockingObserve()
-        verify(commandObserver, atLeastOnce()).onChanged(commandCaptor.capture())
-        assertEquals(Command.UpdateTheme, commandCaptor.firstValue)
+            cancelAndConsumeRemainingEvents()
```

Comment:
would it make sense to extract this somehow? seems easy to forget and it's used in every single test